diff_general/Makefile

```diff
@@ -0,0 +1,2 @@
+all:
+	zip -r autograder.zip *
```

Comment:
I'd prefer to do list the files included a little more explicitly like in https://github.com/gradescope/autograder_samples/blob/master/python/src/make_autograder.sh

Otherwise this will pick up random other stuff that you might not want or need, and might try to put the zipfile inside itself or something.

I also slightly prefer using a shell script instead of a Makefile for this, or at least, if using a Makefile it would be better to define dependencies. I haven't used Make in a little bit but I'm thinking that if you run `make` twice, it won't update things that were changed unless you explicitly list the dependencies for the target? For that reason I'd prefer just making a shell script 1) for similarity with the other example directory, and 2) just to keep things simple.